composer.json

```diff
@@ -22,10 +22,13 @@
     "bitwasp/bech32": "^0.0.1",
     "brick/math": "^0.11",
     "kornrunner/keccak": "^1.1",
-    "illuminate/collections": "^9.0 || ^10.0"
+    "illuminate/collections": "^9.0 || ^10.0",
+    "guzzlehttp/guzzle": "^7.5",
+    "nesbot/carbon": "^2.66"
```

Comment:
As previously, perhaps add a few GitHub details in the PR description?

Reply:
done ✅